internal/cmd/root/root.go

```diff
@@ -14,12 +17,32 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type option struct {
```

Comment:
apologies, i missed this earlier. For consistency, can you rename this `options`?

Reply:
Done.